File: 백준/Bronze/1546. 평균/평균.cc
#include <stdio.h>

int main(void) {
    int n, max = 0;
    double score = 0;
    scanf("%d", &n);
    int arr[n];
    
    for(int i = 0; i < n; i++) {
        scanf("%d ", &arr[i]);
        score += arr[i];
        if(arr[i] > max)
            max = arr[i];
    }
    score /= n;
    double answer = (score/max)*100;
    printf("%lf",answer);
}

File: 백준/Bronze/1546. 평균/평균.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {
        Scanner scanner = new Scanner(System.in);

        int n = scanner.nextInt();
        int[] arr = new int[n];
        double score = 0;
        int i, max = 0;

        for (i = 0; i < n; i++) {
            arr[i] = scanner.nextInt();
            score += arr[i];
            if(arr[i] > max)
                max = arr[i];
        }
        score /= n;
        System.out.println((score/max)*100);
    }
}